lib/ansible/modules/packaging/os/rhsm_repository.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/python
+
+# Giovanni Sciortino (@giovannisciortino)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: rhsm_repository
+short_description: Manage RHSM repositories using the C(subscription-manager)
```

Comment:
Put this on a single line. Not sure if we want C(...) in the short_description.

Reply:
Fixed in the last commit